CONTRIBUTING.md

```diff
@@ -71,3 +81,13 @@ Any documentation can go here. Many apps just use their README.md here.
 - Has a license
 - Has a code of conduct
 - Has someone willing to at least minimally maintain them for the near future
+
+
+## Bots
```

Comment:
Bots -> Apps given our former call discussion.

Although on a larger picture note, I'm not sure I feel that listing this out here is particularly necessary, especially given it won't affect _most_ (I hope) contributions/issues.

Reply:
I feel that if you're going to be using automation that may affect how users perceive the community and the worth of their contributions, it's worth spending a bit of time explaining what you're using and why. This isn't for most people; it's for the few who wonder what happened and didn't expect a bot to respond.